src/Profiler/DatadogProfiler.php

```diff
@@ -21,10 +21,19 @@ class DatadogProfiler implements ProfilerInterface
     private ?Scope $scope = null;
 
     private LoggerInterface $logger;
+    private ?float $sampleRate;
 
     public function __construct(LoggerInterface $logger)
     {
         $this->logger = $logger;
+
+        $sampleRateString = getenv('DD_TRACE_SAMPLE_RATE');
+        if(is_numeric($sampleRateString)) {
+            $sampleRate = floatval($sampleRateString);
+            $this->sampleRate = $sampleRate;
+        } else {
+            $this->sampleRate = null;
```

Comment:
I think the default should be 1 so that all requests get sampled.

Reply:
Done